qutebrowser/browser/cookies.py

```diff
@@ -68,19 +68,27 @@ class CookieJar(RAMCookieJar):
         _lineparser: The LineParser managing the cookies file.
     """
 
-    def __init__(self, parent=None):
+    def __init__(self, parent=None, line_parser=None):
```

Comment:
That should probably be a keyword-only argument (which I plan to use more in general):

```python
def __init__(self, parent=None, *, line_parser=None):
```